libs/react-helpers/src/lib/grid/price-cell.tsx

```diff
@@ -1,37 +1,44 @@
-import React from 'react';
+import { memo, forwardRef } from 'react';
 import { getDecimalSeparator } from '../format';
 export interface IPriceCellProps {
   value: number | bigint | null | undefined;
   valueFormatted: string;
   testId?: string;
 }
 
-export const PriceCell = React.memo(
-  ({ value, valueFormatted, testId }: IPriceCellProps) => {
-    if (
-      (!value && value !== 0) ||
-      (typeof value === 'number' && isNaN(Number(value)))
-    ) {
-      return <span data-testid="price">-</span>;
+export const PriceCell = memo(
+  forwardRef<HTMLSpanElement, IPriceCellProps>(
+    ({ value, valueFormatted, testId }: IPriceCellProps, ref) => {
+      if (
+        (!value && value !== 0) ||
```

Comment:
this whole messy condition can be probably replaced by  `isNumeric` from `react-helpers `